test/test_retriever.py

```diff
@@ -149,10 +180,55 @@ def test_format_insert_value_string_simple():
 
 def test_format_insert_value_string_complex():
     """Test formatting of values for insert statements"""
-    assert test_engine.format_insert_value('my notes: "have extra, stuff"', 'char') == '\'my notes: \\"have extra, stuff\\"\''
+    assert test_engine.format_insert_value('my notes: "have extra, stuff"',
+                                           'char') == '\'my notes: \\"have extra, stuff\\"\''
 
 
 def test_getmd5():
-    """Test md5 sum calculation"""
+    """Test md5 sum calculation given a line"""
+    lines = ['a,b,c\n', '1,2,3\n', '4,5,6\n']
+    assert getmd5(data=lines, data_type='lines') == '0bec5bf6f93c547bc9c6774acaf85e1a'
+
+
+def test_getmd5():
+    """Test md5 sum calculation given a path to data source"""
```

Comment:
The docstring doesn't match the code here and the code is the same as in the function above.